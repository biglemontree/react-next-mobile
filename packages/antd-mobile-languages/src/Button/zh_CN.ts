export default {
  loadingText: '正在加载',
}
