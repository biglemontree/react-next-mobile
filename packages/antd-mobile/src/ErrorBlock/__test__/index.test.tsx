import * as React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Button from '../../Button'
import { ErrorBlock } from '..'

describe('ErrorBlock', () => {
  it('ErrorBlock with error/button/title snapshot', () => {
    const component = shallow(
      <ErrorBlock
        title="antd-mobile"
        error={{
          src:
            'https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*x1LaT6Pn5MwAAAAAAAAAAABkARQnAQ',
          text: '请稍等哦，马上出来',
        }}
        data-y="111"
      >
        <Button type="ghost" capsuleSize="lg" capsule>
          胶囊按钮
        </Button>

        <Button type="ghost" capsuleSize="lg" capsule>
          胶囊按钮
        </Button>
      </ErrorBlock>,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })

  it('ErrorBlock with error/button snapshot', () => {
    const component = shallow(
      <ErrorBlock
        error={{
          src:
            'https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*x1LaT6Pn5MwAAAAAAAAAAABkARQnAQ',
          text: '请稍等哦，马上出来',
        }}
      >
        <Button type="ghost" capsuleSize="lg" capsule>
          胶囊按钮
        </Button>
      </ErrorBlock>,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })

  it('ErrorBlock with error snapshot', () => {
    const component = shallow(
      <ErrorBlock
        error={{
          src:
            'https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*bp-yRoHpmj4AAAAAAAAAAABkARQnAQ',
          text: '页面遇到一些小问题',
        }}
      />,
    )

    expect(toJSON(component)).toMatchSnapshot()
  })
})
