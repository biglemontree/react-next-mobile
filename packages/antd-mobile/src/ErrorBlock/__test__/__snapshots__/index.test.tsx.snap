// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBlock ErrorBlock with error snapshot 1`] = `
<div
  className="amd-error-block"
>
  <img
    className="amd-error-block-poster"
    src="https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*bp-yRoHpmj4AAAAAAAAAAABkARQnAQ"
  />
  <div
    className="amd-error-block-text"
  >
    页面遇到一些小问题
  </div>
</div>
`;

exports[`ErrorBlock ErrorBlock with error/button snapshot 1`] = `
<div
  className="amd-error-block"
>
  <img
    className="amd-error-block-poster"
    src="https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*x1LaT6Pn5MwAAAAAAAAAAABkARQnAQ"
  />
  <div
    className="amd-error-block-text"
  >
    请稍等哦，马上出来
  </div>
  <div
    className="amd-error-block-content"
  >
    <Button
      capsule={true}
      capsuleSize="lg"
      type="ghost"
    >
      胶囊按钮
    </Button>
  </div>
</div>
`;

exports[`ErrorBlock ErrorBlock with error/button/title snapshot 1`] = `
<Container
  title="antd-mobile"
>
  <div
    className="amd-error-block amd-error-block-title"
    data-y="111"
  >
    <img
      className="amd-error-block-poster"
      src="https://gw.alipayobjects.com/mdn/rms_997765/afts/img/A*x1LaT6Pn5MwAAAAAAAAAAABkARQnAQ"
    />
    <div
      className="amd-error-block-text"
    >
      请稍等哦，马上出来
    </div>
    <div
      className="amd-error-block-content"
    >
      <Button
        capsule={true}
        capsuleSize="lg"
        type="ghost"
      >
        胶囊按钮
      </Button>
      <Button
        capsule={true}
        capsuleSize="lg"
        type="ghost"
      >
        胶囊按钮
      </Button>
    </div>
  </div>
</Container>
`;
